rockcraft/services/lifecycle.py

```diff
@@ -27,6 +27,7 @@
 from rockcraft.models.project import Project
 
 # Enable the craft-parts features that we use
+Features.reset()
```

Comment:
I'm curious what drove this change. Nothing wrong with it, but I'm wondering if you uncovered a bug in craft-application?

Reply:
There is a bug but I think only happens in tests. The issue is that a fixture is enabling the overlay feature, and then _later_ this module gets imported and tries to re-enable it. The singleton then complains that the `Features` class has already been instantiated. We can fix that in a number of ways but I think it's low-ish priority now